changelog.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 2024-07-11 - Version 4.1.0
```

Comment:
I think one other change made it in after version 4.0.4

Reply:
Will address with a rebase.